pkgs/by-name/go/golangci-lint/package.nix

```diff
@@ -1,13 +1,13 @@
 {
-  buildGoModule,
+  buildGoLatestModule,
```

Comment:
Please add a comment why the package requires that version and why it can't use buildGoLatestModule. Comment should also state to check the release notes on update for indication to bump this.